jetty9-compat-base/src/main/java/com/google/apphosting/runtime/jetty9/SessionManager.java

```diff
@@ -196,7 +196,8 @@ public void save() {
 
     protected void save (boolean force)
     {
-      logger.fine("Session " + getId() + "is"+(dirty?" dirty":" not dirty")+(force || dirty ? " saving":" not saving"));
+      if (logger.isLoggable(Level.FINE))
+        logger.fine("Session " + getId() + "is"+(dirty?" dirty":" not dirty")+(force || dirty ? " saving":" not saving"));
```

Comment:
I inherited the "" + "" style formatting.  Will change to more consistent String.format